components/backendappgateway/local.tf

```diff
@@ -1,41 +1,23 @@
 locals {
-  criticality = {
-    sbox     = "Low"
-    aat      = "High"
-    prod     = "High"
-    demo     = "Medium"
-    ithc     = "Medium"
-    ldata    = "Medium"
-    perftest = "Medium"
-    preview  = "Medium"
-    dev      = "Medium"
-    test     = "Medium"
-    stg      = "Medium"
-  }
 
-  env_display_names = {
-    sbox     = "Sandbox"
-    aat      = "Staging"
-    prod     = "Production"
-    demo     = "Demo"
-    ithc     = "ITHC"
-    ldata    = "LDATA"
-    perftest = "Test"
-    preview  = "Dev"
-    dev      = "Dev"
-    test     = "Test"
-    stg      = "Staging"
+  log_analytics_env_mapping = {
```

Comment:
why is this added here? it wasn't here before

Reply:
There is a reference but can't find the relevant code. Bit puzzling ...
subscription_id = local.log_analytics_workspace[[for x in keys(local.log_analytics_env_mapping) : x if contains(local.log_analytics_env_mapping[x], var.env)][0]].subscription_id
}